### DISCUSSION TITLE
Way to detect unused snapshots and remove them?

### DISCUSSION THREAD

**Discussion by vitorvarela** [pointfreeco/swift-snapshot-testing#619] [Q&A]
Hi Folks,

When running a full test suite, is there a way to detect which images are not being used? 
I imagine there to be a flag that you can pass in to clean up unused snapshots.

We are running into scenarios where engineers are removing or renaming tests but not removing the related snapshots -- it would be nice to be able to automate this.

**Comment by Sherlouk** [comment]
There was a PR for this a while back but I believe it was closed because it would give false positives if you have a test suite which spans multiple platforms (like this projects tests!)
